### PR TITLE
fr: Removes duplicated frontmatter keys (learn folder)

### PR DESCRIPTION
**Pull request by Lou8is**
### Description

Removes duplicated frontmatter keys (see #7412) from the **files/fr/learn** sub-folder.

Only `title`, `short_title`, `slug` and` l10n.*` are left, essentially removed `tags`, `original_slug`, `browser-compat`, `file-type`.

### Motivation

Help keeping the FR translation nice and up-to-date :)

### Related issues and pull requests

Relates to #7412 
Relates to 6 other PR for the other subfolders (#13629, #13630, #13631, #13632, #13633, #13664)

### CODE REVIEW

File: files/fr/learn/forms/how_to_build_custom_form_controls/index.md [hunk 1]
```diff
@@ -1,15 +1,7 @@
...
-tags:
-  - Avancé
-  - Exemple
-  - Formulaires
-  - Guide
-  - HTML
-  - Web
 translation_of: Learn/Forms/How_to_build_custom_form_controls
-original_slug: >-
   Web/Guide/HTML/Formulaires/Comment_construire_des_widgets_de_formulaires_personnalisés
```
Comment (SphinxKnight) [inline files/fr/learn/forms/how_to_build_custom_form_controls/index.md:5]:
```suggestion
```

**Review by SphinxKnight — CHANGES_REQUESTED** [review]
multi-line metadata should be properly removed.
As far as I can see (I searched for `: >-`): 

- `/files/fr/learn/forms/how_to_build_custom_form_controls/index.md`
- `/files/fr/learn/forms/how_to_build_custom_form_controls/example_1/index.md`
- `/files/fr/learn/forms/how_to_build_custom_form_controls/example_2/index.md`
- `/files/fr/learn/forms/how_to_build_custom_form_controls/example_3/index.md`
- `/files/fr/learn/forms/how_to_build_custom_form_controls/example_4/index.md`
- `/files/fr/learn/forms/how_to_build_custom_form_controls/example_5/index.md`
- `/files/fr/learn/javascript/objects/adding_bouncing_balls_features/index.md`

need to be edited.

**Comment by SphinxKnight** [comment]
@Lou8is thanks that was fast :) (I was working on it locally as well)

**Comment by Lou8is** [comment]
I had time and saw the notification so... 😄 
However I am not sure how to handle the PR Test fail? I do not understand whats wrong (I'd guess the {{LearnSidebar}} macro).

**Comment by SphinxKnight** [comment]
> I had time and saw the notification so... 😄 However I am not sure how to handle the PR Test fail? I do not understand whats wrong (I'd guess the {{LearnSidebar}} macro).

thanks for this (and for all the work on this PR and the others)

I think it has to do with the content used from the subpages in the `EmbedLiveSample` calls. I'm investigating and comparing with en-US to be sure.

**Comment by SphinxKnight** [comment]
I'll be back at it later tonight or tomorrow :)

**Comment by Lou8is** [comment]
Well removing EmbedLiveSample macros from /fr/docs/Learn/Forms/How_to_build_custom_form_controls also removes the errors so you were right. However, i compared that with the EN content, and it does not seems to be any difference regarding that.

I suspect the error comes from arguments being deprecated in the EmbedLiveSample macro (see line 17 of https://github.com/mdn/yari/blob/main/kumascript/macros/EmbedLiveSample.ejs).

That might be a whole other issue to rewrite pages so they don't call out-of-page code samples.

**Comment by cw118** [comment]
It seems that the EmbedLiveSample macros on the en-US page need to be updated as well. I opened [an issue on mdn/content](https://github.com/mdn/content/issues/27801) to confirm and I am preparing a pull request (for en-US).

**Comment by cw118** [comment]
The accepted solution on en-US (cf. mdn/content#27813) was to copy the code of each EmbedLiveSample from the `Learn/Forms/How_to_build_custom_form_controls/Example_1`, `Learn/Forms/How_to_build_custom_form_controls/Example_2` etc. pages and add it to the `Learn/Forms/How_to_build_custom_form_controls` file above each corresponding EmbedLiveSample using hidden codeblocks (html hidden, css hidden. js hidden).

Some comments (Styles requis, etc.) were removed from the hidden codeblocks (not from the `Learn/Forms/How_to_build_custom_form_controls/Example` pages!), and links pointing to the source code need to be moved as well.

See the [pull request diff on mdn/content](https://github.com/mdn/content/pull/27813/files) for more details, and let us know if help is needed for fixing the live samples :)

**Comment by Lou8is** [comment]
I've fixed that page as you demonstrated and did not thought about looking into the other pages.

Seems like these files are also impacted, at list in the FR translation. I'm working on them but listing them if you'd like to check the EN content.

(Impacted macros per page)

- [ ] (1) /fr/learn/forms/how_to_structure_a_web_form/index.md
- [ ] (1) /fr/learn/forms/your_first_form/index.md
- [ ] (3) /fr/learn/html/introduction_to_html/advanced_text_formatting/index.md
- [ ] (2) /fr/learn/html/multimedia_and_embedding/images_in_html/index.md
- [ ] (1) /fr/learn/html/multimedia_and_embedding/other_embedding_technologies/index.md
- [ ] (1) /fr/learn/html/tables/advanced/index.md
- [ ] (2) /fr/learn/javascript/building_blocks/conditionals/index.md
- [ ] (2) /fr/learn/javascript/building_blocks/events/index.md
- [ ] (2) /fr/learn/javascript/first_steps/a_first_splash/index.md

**Comment by Lou8is** [comment]
Some macros use "hide-codepen-jsfiddle" as parameter $6. Removing it does not seem to have functional impact, however it changes the page display (more margins or something). Any advice on that?

**Comment by cw118** [comment]
> Some macros use "hide-codepen-jsfiddle" as parameter $6. Removing it does not seem to have functional impact, however it changes the page display (more margins or something). Any advice on that?

Parameter $6 is also deprecated. However, some say that it was [incorrectly/prematurely deprecated](https://github.com/mdn/content/issues/25005#issuecomment-1460687579), so MDN might keep this parameter. 

In case MDN does keep parameter $6 (perhaps also [parameter $5](https://github.com/mdn/content/issues/25005#issuecomment-1460859179)) in the future, I would say if the `en-US` page uses it and all checks pass then we can keep the parameter. Not 100% sure though. 

@SphinxKnight any thoughts regarding parameters $5 and $6 of EmbedLiveSample?

**Comment by cw118** [comment]
> Edit: checks passed, maybe I did not modified those files. I could modified them in this PR or in a separate one.

Since this PR already modifies a lot of files/lines and the scope is not to fix EmbedLiveSample macros (and clarification is needed regarding parameters $5 and $6), let's remove deprecated EmbedLiveSample parameters in a separate one.

**Comment by SphinxKnight** [comment]
> > Edit: checks passed, maybe I did not modified those files. I could modified them in this PR or in a separate one.
> 
> Since this PR already modifies a lot of files/lines and the scope is not to fix EmbedLiveSample macros (and clarification is needed regarding parameters $5 and $6), let's remove the remaining deprecated EmbedLiveSample parameters in a separate one.

Agreed :) 
Sorry for the late reaction here, @Lou8is please let us know if you need help spltting the EmbedLiveSample work on another PR.

**Comment by SphinxKnight** [comment]
For args $5 and $6 of `EmbedLiveSample`, I never used them (or cannot remember) and since https://github.com/mdn/content/issues/25005 is kind of a deep dive into KumaScript arcanas and apparently some stuff is private :( 

In other words: looks over-complicated and sticking with what en-US does is probably the best solution for now.

**Comment by Lou8is** [comment]
I'll try to do that other PR. For what I've seen, I might first raise an issue to discuss it before doing an actual PR, because from what I already saw, the remaining macros with a $6 arg are not in the EN version + modify how the code is rendered.

**Comment by SphinxKnight** [comment]
I've thought a bit more about this one, esp. now that #14380 suffers from the same issue.
Instead of extanding the scope of this one, I'd suggest the following course of action: 

1. For this PR: reset to https://github.com/mdn/translated-content/pull/13663/commits/a2623857c6e1226f9af56387b76bc9bca1046675
2. Merge this one :) (after a review) 
3. Fix what need to be fixed under /learn regarding EmbedLiveSample calls in #14401

Sorry for this @Lou8is and thanks once more for the help.

**Comment by Lou8is** [comment]
@SphinxKnight hmm not convinced I've managed this one correctly, I might need some help on reverting (or I could just re-do the edits on a clean branch).
+ I've reverted only up to a8514164d3f252eaf83eb9b74436ba8c3c80ca43 as it was corrections to the original modifications.

**Comment by SphinxKnight** [comment]
@Lou8is if you made sure the "newer commits" were on another branch for https://github.com/mdn/translated-content/pull/14401 , locally (assuming you have `#7412-cleans-metadata-learn` as a branch and `origin` as a remote to your GH fork), you can:

```
git checkout #7412-cleans-metadata-learn 
git reset --hard a8514164d3f252eaf83eb9b74436ba8c3c80ca43
git push -f origin #7412-cleans-metadata-learn 
```
Since you have `[fix-embedlivesample-macros-learn](https://github.com/Lou8is/translated-content/tree/fix-embedlivesample-macros-learn)`, the commits for the EmbedLiveSample situation should be kept.
This way we have a "proper" history to review on this PR.

Please let me know if you need help in a "synchronous" way and I'll gladly help :)

**Comment by SphinxKnight** [comment]
since the changes now touch a lot of files (incl. other locales and some engine/core stuff), I'll go ahead for the push

**Comment by Lou8is** [comment]
Yeah thanks for handling that. I obviously messed up with git.
I'll handle the EmbedSampleLive in the ongoing PR :)

File: files/fr/learn/css/building_blocks/box_model_tasks/index.md [hunk 2]
```diff
@@ -1,8 +1,7 @@
 ---
 title: 'Testez vos connaissances : le modèle de boîte'
 slug: Learn/CSS/Building_blocks/Box_Model_Tasks
-l10n:
-  sourceCommit: 40cfeaf2623824ff3acf9d95af67a0498e23e3e8
+translation_of: Learn/CSS/Building_blocks/Box_Model_Tasks
```
Comment (SphinxKnight) [inline files/fr/learn/css/building_blocks/box_model_tasks/index.md:4]:
```suggestion
translation_of: Learn/CSS/Building_blocks/Box_Model_Tasks
l10n:
  sourceCommit: 40cfeaf2623824ff3acf9d95af67a0498e23e3e8
```

**Review by SphinxKnight — CHANGES_REQUESTED** [review]

**Review by SphinxKnight — APPROVED** [review]

**Pull request merged**